actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -14,7 +14,7 @@ class ERB
         class_attribute :erb_implementation, default: Erubi
 
         # Do not escape templates of these mime types.
-        class_attribute :escape_whitelist, default: ["text/plain"]
```

Comment:
Since this is public, don't think we should just blanket change this. We can add

```ruby
[self, singleton_class].each do |base|
  base.alias_method :escape_whitelist, :escape_allowlist
  base.alias_method :escape_whitelist=, :escape_allowlist=

  base.deprecate(
    escape_whitelist: 'use #escape_whitelist instead',
    :escape_whitelist= => 'use #escape_whitelist= instead'
  )
end
```

to deprecate until the next version